lib/zclient.h

```diff
@@ -245,7 +246,8 @@ struct zapi_ipv6
 };
 
 extern int zapi_ipv6_route (u_char cmd, struct zclient *zclient, 
-                     struct prefix_ipv6 *p, struct zapi_ipv6 *api);
+                     struct prefix_ipv6 *p, struct prefix_ipv6 *src_p,
+                     struct zapi_ipv6 *api);
```

Comment:
I would argue that it belongs into the same "scope" as the destination. Since the destination is not in the 'struct zapi_ipv6' but passed individually, I would also pass the source individually, since dst/src really should be treated as a tuple/pair.